ui.R

```diff
@@ -2,99 +2,124 @@ library(shiny)
 library(shinydashboard)
 library(shinyBS)
 library(shinyjs)
+library(DT)
+
+## App Meta Data----------------------------------------------------------------
+APP_TITLE  <<- "Title of the app"
+APP_DESCP  <<- paste(
+  "Description of the app",
+  "In this app the goal is to learn about the reasoning of
+  a hypothesis test about proportions."
+)
+## End App Meta Data------------------------------------------------------------
 
-playerdata <- read.csv("NBA1617E.csv", header = TRUE)
-
+#Data Wrangling
+playerdata <- read.csv(file = "NBA1819.csv", header = TRUE)
 # Filter the player data so that it does not choose a player who has no free throw attempts => no free throw %
 index1 <- which(((playerdata$FTA >= 1) * (playerdata$FTA <= 1000)) == 1)
 playerdata2 <- playerdata[index1, ]
-
 # create a list of just the players names to be selected from later
 PlayerNames <- playerdata2[, 1]
 
 #  "ui file must return UI object", perform ui construction:
 dashboardPage(
-  skin = "blue",
-  dashboardHeader(title = "Hypothesis Testing with NBA data", titleWidth = 300),
-
+  skin = "purple",
+  dashboardHeader(title = "Hypothesis Testing", titleWidth = 250,
+                  tags$li(class = "dropdown", actionLink("info", icon("info"))),
+                  tags$li(class = "dropdown",
+                          tags$a(href='https://github.com/EducationShinyAppTeam/BOAST',
+                                 icon("github"))),
+                  tags$li(class = "dropdown",
+                          tags$a(href='https://shinyapps.science.psu.edu/',
+                                 icon("home")))
+  ),
+  
   # Sidebar
   dashboardSidebar(
-    width = 260,
+    width = 250,
     sidebarMenu(
       id = "tabs",
-      menuItem("Overview", tabName = "overview", icon = icon("dashboard")),
-      menuItem("Filtering", tabName = "filtering", icon = icon("wpexplorer")),
-      menuItem("Hypothesis Testing", tabName = "hypothesis_testing", icon = icon("cogs")),
-      menuItem("Data", tabName = "data", icon = icon("table"))
+      menuItem("Overview", tabName = "Overview", icon = icon("dashboard")),
+      menuItem("Explore", tabName = "Explore", icon = icon("wpexplorer")),
+      menuItem("Challenge", tabName = "Challenge", icon = icon("cogs")),
+      menuItem("Data", tabName = "Data", icon = icon("table")),
```

Comment:
I do not understand what the purpose of displaying this data table is. We need to identify and articulate the purpose of displaying the data table. If we can't/don't, then we need to remove this tab.

If the purpose is to make the data available, then we can link the user to where they can get the data.